### PR TITLE
ci(tags): updating ci

### PR DESCRIPTION
**Pull request by nikola-milosa**

### CODE REVIEW

**Review by sasa-tomic — APPROVED** [review]
I still find it challenging to distinguish git tags vs container tags. Maybe we'll have to be explicit each time which tags we're mentioning.

File: scripts/ghcr-checker.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import hashlib
```
[the remark this answers is not in the document]
Comment (nikola-milosa) [reply scripts/ghcr-checker.py:1]: I will refactor it once it works how I think it should work. Then we can review. This is still work in progress

File: scripts/ghcr-checker.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    subprocess.run(["bazel", "run", target["run_target"], "--", "--tag", target["commit"]], check=True)
+
+
+def get_last_pushed_sha(targets, access_token):
+    repo = git.Repo(".")
+    prefix = "https://ghcr.io/v2/dfinity/dre/"
+    print("Will check following targets:", targets)
+
+    response = requests.get(
+        'https://ghcr.io/token?scope="repository:dfinity/dre:pull"', auth=HTTPBasicAuth("nikola-milosa", access_token)
```
[the remark this answers is not in the document]
Comment (LittleChimera) [reply scripts/ghcr-checker.py]: there should be already a GITHUB_TOKEN in the CI env that is authorized for this request

[the remark this answers is not in the document]
Comment (nikola-milosa) [reply scripts/ghcr-checker.py]: The code changed. But no, that token is not the same if you use HTTP, if you use docker with subprocess then it is the same

**Pull request closed without merging**